### PR TITLE
Mutationobserver Updated tests.js (#397)

### PR DESCRIPTION
**Pull request by sheriff91**
Hopefully fixed all code required to move from expect.js to proclaim.js

### CODE REVIEW

**Comment by sheriff91** [comment]
Error:  SAUCE_USER_NAME and SAUCE_API_KEY must be set in the environment.

Where are these set? Is this also something you guys do? @JakeChampion @triblondon (Sorry about this.)

**Comment by JakeChampion** [comment]
Yes, only users with push/write access to this repo can run the tests on CI annoyingly. I will do this now :-)

**Comment by sheriff91** [comment]
@JakeChampion Does this mean my modifications have worked? The test shows 2 failures when testing on IE9 but no further information, and it timed out?

File: polyfills/MutationObserver/tests.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
 /* Assertion bridge between web components test suite and expectjs */
 
+
 var expectRecord = function (src, model) {
 	Object.keys(model).forEach(function(key) {
-		expect(src[key]).to.eql(model[key]);
+		// proclaim(src[key]).to.eql(model[key]);
+    proclaim(src[key], model[key]);
```
Comment (triblondon) [inline polyfills/MutationObserver/tests.js:7]: There are some indentation issues here, @hipkiss91 could you configure your editor to respect our .editorconfig settings?  cc @JakeChampion

File: polyfills/MutationObserver/tests.js [hunk 2]
```diff
@@ -47,15 +51,18 @@ describe('MutationObserver attributes', function() {
     div.setAttribute('a', 'B');
 
     var records = observer.takeRecords();
-    assert.strictEqual(records.length, 2);
+    // assert.strictEqual(records.length, 2);
+    proclaim.strictEqual(records.length, 2);
```
Comment (triblondon) [inline polyfills/MutationObserver/tests.js:55]: You could remove the old-code comments

**Comment by triblondon** [comment]
@hipkiss91 Jake probably merged this into the MO branch because that's an easy way to get our sauce tests to run on the right fork, and we're comfortable with being trigger-happy on merging into non-master branches.  Looks like following your changes we're failing the ESLint at the moment: https://circleci.com/gh/Financial-Times/polyfill-service/1156.

**Comment by sheriff91** [comment]
@triblondon Thanks for the comments. Apologies on not removing the old code, I will do that now. In regard to the indentation - I use tabs of length 4. What length do you guys prefer? If it is 4, then I imagine I missed a tab, apologies.

**Comment by triblondon** [comment]
@hipkiss91 we use editorconfig (http://editorconfig.org/) so your editor should configure itself automatically, but in short we should be using real tabs everywhere, not spaces.  You can see the inconsistency when you view the diff on Github, assuming your preferred tab size is < 8 spaces.  Some files may be incorrectly indented with spaces - I try and correct them when I can, but the most important thing is that we don't mix two styles in one file :-)

**Comment by sheriff91** [comment]
@triblondon Of course - consistency is king. It should all read as though one person wrote it :) Any thoughts as to why the eslint error might occur? 
Also, once I've fixed it - do I just create another PR?

**Pull request merged**